.eslintrc.json

```diff
@@ -18,6 +18,8 @@
     }
   },
   "rules": {
+    "react/jsx-uses-react": "off",
+    "react/react-in-jsx-scope": "off",
```

Comment:
why?

Reply:
for eslint so that it wont throw red squiggly lines when remove 'react' from import statement. since we upgraded the react, the 'react' in the import statement is inactive.

Reply:
https://reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html#eslint

not a fan of more abstractions, but i ok to off. seems half baked attempt:

```
Note how our original code did not need to import React to use JSX anymore! (But we would still need to import React in order to use Hooks or other exports that React provides.)
```